src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const spider = document.querySelector('.spider');
+
+spider.style.left = '50%';
```

Comment:
Instead of setting the position to 50%, calculate the correct position using the dimensions of the field and the spider image.